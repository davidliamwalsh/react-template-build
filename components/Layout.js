import { Component } from 'react'

import '../scss/application.scss'

class Layout extends Component {
  render () {
    return <div className='c-container'>
      { this.props.children }
    </div>
  }
}

export default Layout