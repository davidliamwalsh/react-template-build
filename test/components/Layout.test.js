import Layout from '../../components/Layout'

describe('<Layout />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Layout />
    )

    expect(component).toMatchSnapshot()
  })
}) 