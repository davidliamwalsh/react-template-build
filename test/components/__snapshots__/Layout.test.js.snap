// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> it renders correctly 1`] = `ShallowWrapper {}`;
